### ISSUE TITLE
Grade Distribution: Don't show My grades info in the left tabular along with Grade line.

### ISSUE THREAD

**Issue by pushyamig** [tl-its-umich-edu/my-learning-analytics#628]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 
The Grades view currently shows student grade info in the side table, `MyGrade`  info should be displayed when the student enables to show the grade. Defaults to not to show.

Also, it doesn't make sense to show the `Show My Grade` for Admin, a simple check that if(Admin): remove the checkbox component should be made. `myla_globals` variable should have that info

Change the checkbox color of to be blue instead of red to be consistent with the Resource view.

## Steps to Reproduce :
1.  Become a student using the django admin too, add a particular user in the course from the admin view
2. The default view will not show My grade line but the tabular data will
3. Login as admin and you will see the `Show My Grade` checkbox.

**Comment by pushyamig** [comment]
Assigning this to @ssciolla for getting started with Myla project.

**Comment by ssciolla** [comment]
@pushyamig I made some progress on this today; see [recent commit](https://github.com/ssciolla/my-learning-analytics/commit/ca7c11582b0216f64b8160faeadcddd19b01eb87) in the issue branch of my fork. Color has been changed, and the My grade row of the table now toggles on and off with the grade line.

I did some research on changing the default, but that seems to get into back end operations (I spoke with @jonespm about this earlier). He is working on some changes related to this, so I'm pausing on solving this for the moment. Currently, I'm of the feeling that the default should be handled by the front end.
